compiler/rustc_hir/src/lang_items.rs

```diff
@@ -38,27 +38,34 @@ macro_rules! expand_group {
 // So you probably just want to nip down to the end.
 macro_rules! language_item_table {
     (
-        $( $variant:ident $($group:expr)?, $name:expr, $method:ident, $target:expr; )*
+        $( $(#[attr:meta])* $variant:ident $($group:expr)?, $name:expr, $method:ident, $target:expr; )*
```

Comment:
Why isn't this working?

```
error: no rules expected the token `doc`
   --> compiler/rustc_hir/src/lang_items.rs:201:5
    |
39  | macro_rules! language_item_table {
    | -------------------------------- when calling this macro
...
201 |     /// Trait injected by `#[derive(PartialEq)]`, (i.e. "Partial EQ").
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no rules expected this token in macro call
```

Reply:
You want `$attr`, so it will be a variable.
```
$ rg attr:meta src/librustdoc/
src/librustdoc/lint.rs
67:    ($(#[$attr:meta])* $name: ident, $level: ident, $descr: literal $(,)?) => {
```

Reply:
Thanks! 🤦